independent-publisher-connectors/My Acclaro/apiProperties.json

```diff
@@ -0,0 +1,37 @@
+{
+  "properties":{
+    "connectionParameters":{
+      "api_key":{
+        "type":"securestring",
+        "uiDefinition":{
+          "constraints":{
+            "clearText":false,
+            "required":"true",
+            "tabIndex":2
+          },
+          "description":"Bearer token to access the API",
+          "displayName":"Authorization",
+          "tooltip":"Get your Bearer token from my.acclaro.com"
+        }
+      }
+    },
+    "policyTemplateInstances":[
+      {
+        "templateId":"setheader",
+        "title":"Set API Key",
+        "parameters":{
+          "x-ms-apimTemplateParameter.name":"Authorization",
+          "x-ms-apimTemplateParameter.value":"Bearer @connectionParameters('api_key')",
+          "x-ms-apimTemplateParameter.existsAction":"override",
+          "x-ms-apimTemplate-policySection":"Request"
+        }
+      }
+    ],
+    "iconBrandColor":"#f0f0f0",
```

Comment:
@msanz-acclaro You are following the template provided for IP Connectors. The default iconBrandColor for all the IP Connectors is "da3b01"

Reply:
@sriyen-msft I am realizing I chose the wrong path, as Acclaro (the company I work for) owns the API service, so this is not an IP Connector but a Custom connector. Please accept my apologies for the inconveniences.